### ISSUE TITLE
Meld

### ISSUE THREAD

**Issue by ShawnieBoy** [magarena/magarena#807]
As the new mechanic in Eldritch Moon may take a bit of working out, at the moment I'll include this in scripts missing akin to transform, with both halves pointing to the melded card and vice versa, if only to allow viewing in card explorer. Of course this will not work in practice.

**Comment by ShawnieBoy** [comment]
The 'Melds with' reminder text is an interesting addition, although it is a reminder text which normally we don't show, in this case it serves an increased importance as there is no way of just looking at the back of the card. Italicising this in Cardbuilder for the oracle would make sense in this case, as would having a Meld card property to display the paired card required.

**Comment by lodici** [comment]
Managing the meld mechanic in a RL game must be a nightmare. Can't see a straight-forward way of implementing in UI as it stands.

**Comment by ShawnieBoy** [comment]
In-game it can just be represented as one card, as it is only one permanent. The problem only arises when it's changing zones.

**Comment by ShawnieBoy** [comment]
Maybe having the other side of the Melded permanent/card displayed like a split card instead (mini-brainwave)

**Comment by lodici** [comment]
Good idea, that would fit nicely with the current UI code for transformables.
